app/views/organisations/_header.html.erb

```diff
@@ -1,5 +1,62 @@
 <% if @organisation.is_no_10? %>
-  <h1 class="organisation__no10-banner organisation__margin-bottom"><%= @header.org.title %></h1>
+  <div class="govuk-width-container">
+    
+    <div class="organisation__no10-header organisation__margin-bottom">
+      <%# To do: add asset here and style and name this container as needed %>
+      <span class="govuk-heading-xl govuk-!-text-align-centre">
+      10
+      </span>
+      
+      <div class="organisation__no10-header__banner govuk-!-margin-bottom-6">
+
+        <%= content_tag(:picture) do %>
+          <%= content_tag(:source, nil, { media: "(min-width: 641px)", srcset: "#{image_path('no10header-desktop.png')}" }) %>
```

Comment:
I don't think `nil` is needed here in theory. I would try removing `nil` but if it doesn't work then we can keep it how it is now.

Reply:
Done, using the tag element instead of content_tag.